### PR TITLE
feat: wrapps storing name in the precache in try

### PR DESCRIPTION
**Pull request by jurajpiar**
Fixes issue on a testnet which seems to fail the process of precache on a strange name resolution.

### CODE REVIEW

File: src/services/rns/rns.precache.ts [hunk 1]
```diff
@@ -57,7 +57,11 @@ export async function processRskOwner (eth: Eth, logger: Logger, contractAbi: Ut
     const name = Utils.hexToAscii('0x' + decodedData.params[2].value.slice(218, decodedData.params[2].value.length))
     const tokenId = Utils.numberToHex(Utils.sha3(name) as string)
     const ownerAddress = rskOwnerEvent.returnValues.to.toLowerCase()
-    await Domain.upsert({ tokenId, name: `${name}.${tld}` })
+    try {
+      await Domain.upsert({ tokenId, name: `${name}.${tld}` })
+    } catch (e) {
+      logger.warn(`Domain name ${name}.${tld} for token ${tokenId} could not be stored.`)
```
Comment (julianmrodri) [inline src/services/rns/rns.precache.ts:63]: I think we should UPSERT with null or empty name, so we dont lose the relationship. Not just log the issue.

Comment (julianmrodri) [reply src/services/rns/rns.precache.ts:63]: I think we are using NULL for domains that we could not retrieve the"readable" name

**Comment by AuHau** [comment]
@jurajpiar I would argue that the commit message should have had "fix" scope (eq. `fix:`)... Just for future...

**Comment by jurajpiar** [comment]
@AuHau yes you are right

**Pull request merged**